internal/connection/spinner.go

```diff
@@ -0,0 +1,94 @@
+package connection
+
+import (
+	"fmt"
+
+	"github.com/charmbracelet/bubbles/spinner"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+const spinnerText = `Connecting to the cluster %s at %s.
+   Check the logs at %s.`
+
+var (
+	currSpinner  = spinner.Dot
+	spinnerStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("#40826D"))
+	helpStyle    = lipgloss.NewStyle().Foreground(lipgloss.Color("241")).Render
```

Comment:
Let's take the colors from the theme if possible.

Reply:
AFAIK there is no build in color types in BubbleTea but commented down. [0712bb6](https://github.com/hazelcast/hazelcast-commandline-client/pull/143/commits/0712bb6234fb499a008c1056f9c43b55a3e2754e)